scripts/world_bank/wdi/worldbank.py

```diff
@@ -272,17 +427,20 @@ def download_indicator_data(worldbank_countries, indicator_codes,
         country_df['ISO3166Alpha3'] = country_code
 
         # Add new row to main datframe.
-        worldbank_dataframe = worldbank_dataframe.append(country_df)
-
+        country_df_list.append(country_df)
+        #worldbank_dataframe = pd.concat([worldbank_dataframe, country_df])
```

Comment:
Pls delete commented line not required any more

Reply:
Deleted